src/test/java/net/snowflake/client/jdbc/SnowflakeDriverLatestIT.java

```diff
@@ -1459,6 +1459,9 @@ public void testUploadWithGCSPresignedUrlWithoutConnection() throws Throwable {
       connection = getConnection("gcpaccount");
       Statement statement = connection.createStatement();
 
+      // set parameter for presignedUrl upload instead of downscoped token
+      statement.execute("alter session set GCS_USE_DOWNSCOPED_CREDENTIAL = false");
+
```

Comment:
NIT: I personally suggest to use `paramProperties` to set it. For example, 
```
      Properties paramProperties = new Properties();
      paramProperties.put("GCS_USE_DOWNSCOPED_CREDENTIAL", false);
      connection = getConnection("gcpaccount", paramProperties);
```

Reply:
Good suggestion, updated